pkg/edgeview/src/basics.go

```diff
@@ -350,26 +355,34 @@ func getBasics() {
 
 	if basics.server != "" {
 		var printed bool
+		var authenStr string
+		if clientAuthType != types.EvAuthTypeUnspecified {
+			authen := "certs"
+			if clientAuthType == types.EvAuthTypeSSHRsaKeys {
+				authen = "ssh-keys"
+			}
+			authenStr = fmt.Sprintf(", (authen %s)", authen)
+		}
 		conts := strings.Split(basics.server, "zedcloud.")
 		if len(conts) == 2 {
 			cont2s := strings.Split(conts[1], ".zededa.net")
```

Comment:
Even if this is only for coloring, it is quite odd to see .zededa.net in the source code. If we need that we should use the content of /config/server as the template for splitting the name since /config/server could contain myzedcloud.example.net and EVE-OS should work the same.

Reply:
Haha, i'll remove this part, we don't have to do coloring.